Documentation/ApiOverview/PasswordHashing/Troubleshooting.rst

```diff
@@ -44,11 +44,11 @@ Solutions
 Recommended: Fix the server side
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-It is highly recommended to run PHP 7.2 with argon2i support.
+It is highly recommended to run PHP 7.2 or above with argon2i support.
 Install a PHP build that supports this or make the project hoster support
-PHP 7.2 or above with argon2i. Usually, the argon library is just not installed
+PHP with argon2i. Usually, the argon2i library is just not installed
```

Comment:
Please also adjust the title of the PR to something meaningful, like "Adjustment in password hashing troubleshooting", so it can be identified better in git history.